src/RubyGosu.cxx

```diff
@@ -11415,8 +11415,10 @@ SWIGEXPORT void Init_gosu(void) {
   rb_define_const(mGosu, "MAJOR_VERSION", SWIG_From_int(static_cast< int >(0)));
   rb_define_const(mGosu, "MINOR_VERSION", SWIG_From_int(static_cast< int >(10)));
   rb_define_const(mGosu, "POINT_VERSION", SWIG_From_int(static_cast< int >(8)));
-  rb_define_singleton_method(mGosu, "VERSION", VALUEFUNC(_wrap_VERSION_get), 0);
-  rb_define_singleton_method(mGosu, "LICENSES", VALUEFUNC(_wrap_LICENSES_get), 0);
+  // rb_define_singleton_method(mGosu, "VERSION", VALUEFUNC(_wrap_VERSION_get), 0);
+  rb_define_const(mGosu, "VERSION", SWIG_From_std_string(static_cast< std::string >(Gosu::VERSION)));
+  // rb_define_singleton_method(mGosu, "LICENSES", VALUEFUNC(_wrap_LICENSES_get), 0);
+  rb_define_const(mGosu, "LICENSES", SWIG_From_std_string(static_cast< std::string >(Gosu::LICENSES)));
```

Comment:
I'm not sure about this. rake swig generate the singleton_method-wrapper thing which didn't work. If I manually insert what I would expect to be generated it works. But I have no idea what to do...